dotCMS/src/main/java/com/dotmarketing/image/focalpoint/FocalPointCache.java

```diff
@@ -0,0 +1,74 @@
+package com.dotmarketing.image.focalpoint;
+
+import java.util.Optional;
+import com.dotmarketing.business.Cachable;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.business.DotCacheAdministrator;
+import com.dotmarketing.business.DotCacheException;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+
+
+public class FocalPointCache implements Cachable {
+
+    private DotCacheAdministrator cache;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'cache' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4761083)